org.flycast.Flycast.yml

```diff
@@ -0,0 +1,36 @@
+app-id: org.flycast.Flycast
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: flycast
+rename-desktop-file: flycast.desktop
+rename-icon: flycast
+finish-args:
+  - --device=all
+  - --filesystem=host:ro
+  - --share=network
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --socket=x11
```

Comment:
If it supports using wayland then it should `--socket=fallback-x11`. In any case it needs `--share=ipc`

Reply:
As far as I know, Wayland is supposed to be supported - though I don't know if it is as stable as X11.

Reply:
point is that with this wayland will be used when available so it should be `--socket=fallback-x11` instead of `--socket=x11` which mean : if host doesn't have wayland, enable X11. It's about minising permissions.

Unless there is a reason to always have X11 as well.

Reply:
Got it, thanks for the explanation! I did the changes, they are already pushed here.